### PR TITLE
Config API update for 1.26

### PR DESCRIPTION
**Pull request by tengqm**
This PR updates the Config API references that are generated using the kubernetes-sigs/reference-docs/genref tool.
The tool has been updated to use the latest sources released with 1.26.

### CODE REVIEW

**Comment by sftim** [comment]
@tengqm - mostly for the benefit of less experienced contributors - I recommend adding a brief description of what these changes are and why we make them. 40 words might well be enough.


However, I already have that context.

/approve

File: content/en/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 1]
```diff
@@ -645,7 +655,7 @@ Default: true</p>
 </td>
 <td>
    <p>cpuCFSQuotaPeriod is the CPU CFS quota period value, <code>cpu.cfs_period_us</code>.
-The value must be between 1 us and 1 second, inclusive.
+The value must be between 1 ms and 1 second, inclusive.
 Requires the CustomCPUCFSQuotaPeriod feature gate to be enabled.
 Default: &quot;100ms&quot;</p>
```
Comment (sftim) [inline content/en/docs/reference/config-api/kubelet-config.v1beta1.md:660]: Aside: this feels a bit off (something to consider fixing _upstream_).

Comment (mengjiao-liu) [reply content/en/docs/reference/config-api/kubelet-config.v1beta1.md:660]: This appears to have been fixed in Kubernetes v1.26.
```
clarify CPUCFSQuotaPeriod values, set the minimum to 1ms
```
https://github.com/kubernetes/kubernetes/commit/d0f9e6dc36fb0f6cfff95988e27eb3796c4e6bce

Comment (tengqm) [reply content/en/docs/reference/config-api/kubelet-config.v1beta1.md:660]: @mengjiao-liu So you mean the generated documentation is up to date, right?

Comment (mengjiao-liu) [reply content/en/docs/reference/config-api/kubelet-config.v1beta1.md:660]: > @mengjiao-liu So you mean the generated documentation is up to date, right?

I mean, upstream fixed this issue in version 1.26, but the auto-generated files are still unfixed, and I'm not sure if the auto-generated files are up to date or if there are other issues.

**Pull request merged**